tests/unit/scala/test_utils_suite.py

```diff
@@ -13,6 +13,7 @@
     TEMPORARY_STRING,
     calculate_checksum,
     deprecated,
+    experimental,
```

Comment:
Is it ok that the python test file is in a "scala" directory?

Reply:
It's just a directory name we use to indicate some tests are ported from scala code :) sorry for the confusion